e2e/suites/tableView.test.ts

```diff
@@ -20,6 +20,7 @@ describe('Table view', () => {
   describe('Table actions', () => {
     it('Should show only selected properties', async () => {
       await page.click('.hidden-columns-dropdown');
+      await page.waitFor(200);
```

Comment:
If we are waiting for `#rw_1_listbox li`, why do we need to wait for 200 ms here?  If the wait for the `li` is not good enough, please, input the correct selector that 100% disambiguates the status of the client, and not this arbitrary wait.